src/test/java/com/axibase/tsd/api/method/csv/CSVUploadTest.java

```
@@ -24,24 +28,26 @@
     private static final String RESOURCE_DIR = "csv_upload";
     private static final String ENTITY_PREFIX = "e-csv-simple-parser";
     private static final String METRIC_PREFIX = "m-csv-simple-parser";
-    public static final String PARSER_NAME = "simple-parser";
+    public static final String[] PARSER_NAMES = {"simple-parser", "simple-parser-iso", "simple-parser-ms"};
 
     @Rule
     public TestName name = new TestName();
 
     @BeforeClass
     public static void installParser() throws URISyntaxException, FileNotFoundException {
-        File configPath = resolvePath(RESOURCE_DIR + File.separator+PARSER_NAME+".xml");
-        boolean success = importParser(configPath);
-        assertTrue(success);
+        for (String parserName : PARSER_NAMES) {
+            File configPath = resolvePath(RESOURCE_DIR + File.separator + parserName + ".xml");
+            boolean success = importParser(configPath);
+            assertTrue(success);
```

Let's throw an exception here, cause it's not the main goal of tests